.lycheeexclude

```diff
@@ -114,3 +115,4 @@ https://a.tile.openstreetmap.org/
 http://www.creedthoughts.gov
 https://media-for-the-masses.theacademyofperformingartsandscience.org/
 https://yarnpkg.com/latest.msi
+https://forum.opensearch.org/t/feedback-experimental-feature-connect-to-external-data-sources/11144
```

Comment:
could you try just https://forum.opensearch.org/ here?

Reply:
Updated to https://forum.opensearch.org/. Link check passed.